src/Qowaiv.Validation.Fluent/Qowaiv.Validation.Fluent.csproj

```diff
@@ -4,12 +4,14 @@
   
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net5.0;net6.0</TargetFrameworks>
```

Comment:
Shouldn't we show the supported frameworks in a LIFO order? Like they did in https://docs.microsoft.com/en-us/dotnet/standard/frameworks

Reply:
I don't think that common or making any difference. This is just the config for the compiler to know to which target frameworks to compile (and to pack for NuGet).

Reply:
Fine by me!